src/main/java/org/candlepin/model/EnvironmentCurator.java

```diff
@@ -236,4 +234,15 @@ public String getEnvironmentIdByName(String ownerId, String envName) {
             .setParameter("consumerId", consumer.getId())
             .getResultList();
     }
+
+    public List<String> getEnvironmentContentUUIDs(String environmentId) {
```

Comment:
This method seems more appropriate for `EnvironmentContentCurator.java`

Reply:
Moved to `EnvironmentContentCurator` class.